aws/template.yaml

```diff
@@ -0,0 +1,25 @@
+Description: Parameter store parameters
+Parameters:
+  EnvironmentName:
+    Default: Dev
+    Type: String
+    AllowedValues:
+      - Dev
+      - Stage
+      - Prod
```

Comment:
I would just suggest to not limit environment names to those three values. We should in general try to make deployment templates universal.